src/lib/molecules/TimePicker/TimePickerPopover.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useRef, useEffect, forwardRef, useImperativeHandle, ReactNode } from 'react';
+
+// Helpers
+//@ts-ignore
+import { useKeyDown, useClickOutside } from 'hooks';
+
+// Components
+import Popover from '../../atoms/PopoverV2';
+
+// Types
+import { ChildRef } from './TimePicker';
+
+interface ITimePickerPopover {
+    toggleHandler: (open: boolean) => boolean | void;
+    children: ReactNode;
+    readOnly: boolean;
+    value: string;
+    positions: ('bottom' | 'top' | 'left' | 'right')[];
```

Comment:
type positions = 'bottom' | 'top' | 'left' | 'right';
   
 positions?: positions[] | positions;

Reply:
We can not use type like this because 'positions' props should be only array